ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -258,20 +258,6 @@ impl RawData for RawDataServerWrapper {
                     {
                         Ok(TransactionsDataStatus::Success(transactions)) => transactions,
                         Ok(TransactionsDataStatus::AheadOfCache) => {
-                            info!(
```

Comment:
once we delete these ahead of cache loggings, how do we track this status anymore?

Reply:
We can also use `sample!` here but I think it would be random which `connection_id` it will log for